src/service/receipt.service.ts

```diff
@@ -29,8 +32,24 @@ import {
 } from 'nem2-sdk';
 
 export class ReceiptService {
+    private transactionStatementsTable: HorizontalTable;
```

Comment:
Maybe the transactionStatementsTable is not needed. We could print the header ``transactionStatements + source``, and then all the related receipts formatted as table individually.

Reply:
I don't understand this clearly. Could you please describe it more clearly?